### PR TITLE
Add an empirical comparison of the different storage types

### LINKED ISSUE #65: Add an empirical comparison between array types
Compare disk/memory use and speed of and add this to the README, to give people an idea of the trade-offs:

* Array
* Memory-mapped array
* Quantized array
* Memory-mapped quantized array.

I have already done parts of this, so I can take this one.

### PR DESCRIPTION
**Pull request by danieldk**
Fixes #65.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,66 @@ of [pretrained embeddings](https://finalfusion.github.io/pretrained) is offered
...
+inside/outside the vocabulary. *Lookup* contains a mixture of known
+and unknown words.
+
+| Storage    | Lookup | Known lookup | Unknown lookup |   Memory |     Disk |
+|:-----------|-------:|-------------:|---------------:|---------:|---------:|
+| array      | 449 ns |       232 ns |          18 μs | 3213 MiB | 3213 MiB |
+| array mmap | 833 ns |       494 ns |          23 μs | Variable | 3213 MiB |
+| opq        |  40 μs |        21 μs |         962 μs |  402 MiB |  402 MiB |
+| opq mmap   |  41 μs |        21 μs |         960 μs | Variable |  402 MiB |
+
```
Comment (sebpuetz) [inline README.md]: µ and n are similar enough that I just spent a good minute wondering how on earth `opq` could be 10 times faster than `array`.

Comment (danieldk) [reply README.md]: Could that be a font issue? For me they look really different.

Screenshot from 2019-10-01 13-46-22

Comment (sebpuetz) [reply README.md]: Rather a "not-paying-attention" issue. I was focusing on the numbers, apparently my mind didn't expect to encounter different units.

Comment (sebpuetz) [reply README.md]: image

Comment (danieldk) [reply README.md]: Maybe we could add speedups compared to the best number, let me try how that looks.

Comment (danieldk) [reply README.md]: Too busy, I have now added a note to warn for unit differences.

**Pull request merged**